hydrabus_framework/hbfconsole

```diff
@@ -1,5 +1,10 @@
+#!/usr/bin/env python
+
 import configparser
-from core.engine import HydraFramework
```

Comment:
Same here. Empty space between system import and your own

Reply:
Done [#4c395b4](https://github.com/hydrabus-framework/framework/commit/4c395b4dcea89f1c7d4ca197603981b491b7325e)